gateway/src/apicast/balancer.lua

```diff
@@ -70,6 +70,24 @@ local function set_timeouts(balancer, timeouts)
   end
 end
 
+local function set_balancer_retry(balancer)
+  local _, err = balancer:set_more_tries(1)
```

Comment:
Wouldn't it be nicer as `balancer:retry_next_request()` ?  Wrapping the OpenResty API into something more human-friendly.

Reply:
Good idea :+1:
Changed.